extensions/usememos/package.json

```diff
@@ -0,0 +1,102 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "usememos",
+  "title": "Usememos",
+  "description": "Usememos is an extension for usememos/memos",
+  "icon": "icon.png",
+  "owner": "bone-corp",
```

Comment:
Can we remove this so it's release under your name instead?

Reply:
Of course.